mediapipe/examples/android/src/java/com/google/mediapipe/apps/instantmotiontracking/MainActivity.java

```diff
@@ -141,7 +141,9 @@
   // last time the GIF was updated
   private long gifLastFrameUpdateMS = System.currentTimeMillis();
   private Bitmap defaultGIFTexture = null; // Texture sent if no gif available
-  private final String DEFAULT_GIF_TEXTURE = "default_gif_texture.bmp";
+  private final String GIF_ASPECT_RATIO_TAG = "gif_aspect_ratio";
+  private float gifAspectRatio = 1.0f; // GIF width:height
```

Comment:
nit: while we're here, let's reorder these so they're a little easier to read-- maybe bring all the constants (final) up above the non-constants?